src/scenic/simulators/robosuite/model.scenic

```diff
@@ -0,0 +1,253 @@
+# src/scenic/simulators/robosuite/model.scenic
+
+"""Scenic world model for RoboSuite - Custom Environments Only."""
+
+from .simulator import RobosuiteSimulator, SetJointPositions, OSCPositionAction
+
+# Global parameters with defaults matching Robosuite's defaults
+param env_config = {}
+param controller_config = None
+param camera_view = None
+param render = True
+param real_time = True
+param speed = 1.0
+param lite_physics = None  # None = use Robosuite default (True)
+
+# Simulator - no more use_environment parameter
+simulator RobosuiteSimulator(
+    render=globalParameters.render,
+    real_time=globalParameters.real_time,
+    speed=globalParameters.speed,
+    env_config=globalParameters.env_config,
+    controller_config=globalParameters.controller_config,
+    camera_view=globalParameters.camera_view,
+    lite_physics=globalParameters.lite_physics
+)
+
+# Default values dictionary
+DEFAULTS = {
```

Comment:
I don't think gathering all the defaults into a dict like this makes the code any clearer. I'd just put the values directly into the classes where they are used, so that they're immediately present if you go to look up the default value of a property in the class definition.